idl/Microsoft.UI.Xaml.idl

```diff
@@ -169,5 +178,14 @@ namespace MU_XC_NAMESPACE
 
             static Windows.UI.Xaml.DependencyProperty ControlsResourcesVersionProperty{ get; };
         }
+
+        // To be removed
+        [MUX_PREVIEW]
```

Comment:
I see the ControlsResourcesVersion above is still in MUX_PREVIEW. Is that intentional @ranjeshj ? I thought that was API reviewed.

Regardless for this whole change you'll want to compare the winmd from the last 2.6 release with the one from here and look at the XamlControlsResources type. Make sure the previous interfaces are the same UUID as before. They don't have to be the same name exactly (like if it changes from IXamlControlsResources2 to IXamlControlsResources3, that's fine).

Reply:
I believe so. you can mark it as public. @MikeHillberg as FYI.